test/test_rosbag/bag_migration_tests/CMakeLists.txt

```diff
@@ -97,3 +97,7 @@ configure_file(test/random_play_sim.xml.in
   ${PROJECT_BINARY_DIR}/test/random_play_sim.xml)
 add_rostest(${PROJECT_BINARY_DIR}/test/random_play_sim.xml
   DEPENDENCIES run_tests_test_rosbag_rostest_test_random_record.xml)
+
+# Make sure that rostest random_record.xml is not executed twice, running in
+# parallel (https://github.com/ros/ros_comm/pull/1651#issuecomment-482148146):
+add_dependencies(run_tests_test_rosbag_rostest_test_random_play.xml run_tests_test_rosbag_rostest_test_random_play_sim.xml)
```

Comment:
This change is already present on the target branch. Please remove it from this PR.

Reply:
Sorry, I did not notice that you force-pushed another version of my patch on branch `fix-1343-melodic-devel` (from #1651). I now cherry-picked the new commit on top of that and force-pushed it to here. I assume that is what you meant.